go.mod

```diff
@@ -2,17 +2,17 @@ module github.com/asecurityteam/serverfull
 
 go 1.22
 
-toolchain go1.23.1
+toolchain go1.24.3
 
 require (
-	github.com/asecurityteam/logevent/v2 v2.0.1
-	github.com/asecurityteam/runhttp v0.6.5
-	github.com/asecurityteam/settings/v2 v2.0.1
-	github.com/aws/aws-lambda-go v1.47.0
-	github.com/go-chi/chi v1.5.5
+	github.com/asecurityteam/logevent/v2 v2.0.2
+	github.com/asecurityteam/runhttp v0.6.6
+	github.com/asecurityteam/settings/v2 v2.0.3
+	github.com/aws/aws-lambda-go v1.48.0
+	github.com/go-chi/chi/v5 v5.2.1
```

Comment:
@gcase555 do we know if v5 is backwards compatible? I am not 100% sure the tests cover this.